app/models/concerns/slack_message_creation.rb

```diff
@@ -0,0 +1,69 @@
+module SlackMessageCreation
+  extend ActiveSupport::Concern
+
+  private
+
+  def create_message
```

Comment:
@sojan-official is it okay for a method that is supposed to be an endpoint be in a concern?

Reply:
@scmmishra This is not an endpoint. It is used in the background job.

Reply:
However, I believe this concern is not written is great way. I will merge this so that the bug is fixed. Please re-consider the implmentation @muhsin-k

Reply:
Sure. I will pair with @sojan-official  and create separate PR.

Reply:
@muhsin-k  models/concerns is suited for placing methods we share across different models. 
Example: `avatarable`, `reauthorizable` etc. Since we can share this code across multiple models. 

In this particular case, the logic placed here is tied very tightly to slack message builder. 
We can place this module alongside incoming_message_builde itself in the same folder .. as we are moving this method into a helper module specifically to satisfy the code climate requirement.  also we don't need to extend `activesupportconcern`

Reply:
* another option is app/helpers . but since i don't forsee us reusing this anywhere. lets keep it alondside the incoming message builder .. under integrations/slack namespace

Reply:
https://github.com/chatwoot/chatwoot/pull/7912